### ISSUE TITLE
Starting from version 1.1.0 error when starting a task

### ISSUE THREAD

**Issue by SineYlo** [pekeq/gulp-libsquoosh#14]
Hi!
When starting a task, it knocks out such an error:
```
Error: Cannot find module 'C:\C:\Users\vavsh\Desktop\temp\node_modules\@squoosh\lib\build\avif_node_enc_mt.worker-be931951.js'
    at Function.Module._resolveFilename (node:internal/modules/cjs/loader:933:15)
    at Function.Module._load (node:internal/modules/cjs/loader:778:27)
    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:81:12)
    at MessagePort.<anonymous> (node:internal/main/worker_thread:187:24)
    at MessagePort.[nodejs.internal.kHybridDispatch] (node:internal/event_target:562:20)
    at MessagePort.exports.emitMessage (node:internal/per_context/messageport:23:28)
```
My task code:
```
import { src, dest } from 'gulp';
import squoosh from 'gulp-libsquoosh';
import rename from 'gulp-rename';
import { config } from '../config';

const imageOptimizationJpg = () => {
  return src(config.source.convertjpg)
    .pipe(rename({
      extname: '.jpg',
    }))
    .pipe(squoosh({
      mozjpeg: {},
      webp: {},
      avif: {},
    }))
    .pipe(rename({
      dirname: 'pictures/',
    }))
    .pipe(dest(config.build.pictures));
};
```
The funny thing is that this problem does not exist on version — `1.0.12`
And all subsequent versions after this one go with an error

**Comment by pekeq** [comment]
Hello,

This issue seems to be a known bug in Squoosh on windows.
https://github.com/GoogleChromeLabs/squoosh/issues/1119

This bug has been fixed on the issue, but @squoosh/lib 0.4.0 release still contains this bug. (May be fixed on future release build.)

Because of gulp-libsquoosh 1.1.x depends on @squoosh/lib 0.4.0,  you cannot use gulp-libsquoosh 1.1.x on Windows currently.

To avoid this issue, you can use gulp-libsquoosh 1.0.12.
This version depends on @squoosh/lib 0.3.1 which does not include this bug.

**Comment by SineYlo** [comment]
Understood, thanks for the answer!
In general, we have to wait until they release a fix for this problem.
